bin/plugin/commands/changelog.js

```diff
@@ -757,6 +757,76 @@ function sortFeatureGroups( featureGroups ) {
 	);
 }
 
+/**
+ *
+ * @param {IssuesListForRepoResponseItem[]} pullRequests List of pull requests.
+ *
+ * @return {string} The formatted changelog string.
+ */
+function formatContributors( pullRequests ) {
```

Comment:
If folks are happy with the general direction I'm happy to refactor this a bit and DRY up the methods.